regression/incremental/unsat_core.smt2.expected.out

```diff
@@ -0,0 +1,3 @@
+sat
+unsat
+( c17 c15 )
```

Comment:
I would suggest to also add `unsat_core_global` variant with additional `(set-option :global-declarations true)`.
Currently, this returns `( c0 c15 )`, which is certainly wrong. It should still return `( c17 c15 )` right?